.github/workflows/build-deploy-doc.yml

```diff
@@ -0,0 +1,53 @@
+name: Build and Deploy documentation site
+
+on: [push]
+
+jobs:
+  build-and-deploy:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v4
+
+    - name: Checkout Endo
+      uses: actions/checkout@v4
+      with:
+        repository: endojs/endo
+        path: endo
+
+    - name: Build Endo docs
+      run: |
+        cd endo
+        yarn add -D typedoc-plugin-markdown -W
+        yarn typedoc --plugin typedoc-plugin-markdown
+
+    - name: Move Endo docs into main
+      run: mv endo/api-docs main/reference/endo
+
+    - name: Checkout agoric-sdk
+      uses: actions/checkout@v4
+      with:
+        repository: agoric/agoric-sdk
+        path: agoric-sdk
+
+    - name: Build agoric-sdk docs
+      run: |
+        cd agoric-sdk
+        yarn add -D typedoc-plugin-markdown -W
+        yarn typedoc --plugin typedoc-plugin-markdown
```

Comment:
looks like this expects a `yarn build` before it.

```
packages/governance/test/swingsetTests/committeeBinary/test-committee.js:6:23 - error TS2307: Cannot find module '@agoric/zoe/bundles/bundle-contractFacet.js' or its corresponding type declarations.

6 import zcfBundle from '@agoric/zoe/bundles/bundle-contractFacet.js';
```

Reply:
I wonder how much this slows down ci.

Reply:
Seems to be [around 1 minute](https://github.com/Agoric/documentation/actions/runs/8442634353/job/23124473938?pr=1038#step:7:218) for the `yarn build` command